app/models/user.rb

```diff
@@ -25,10 +25,13 @@ class User < ApplicationRecord
   has_many   :reviewable_grants,      through: :grant_reviewers,
                                       source: :grant
 
-  has_many   :submissions,            class_name: 'GrantSubmission::Submission',
-                                      foreign_key: :created_id,
+
+  has_many   :submission_applicants,   class_name: 'GrantSubmission::SubmissionApplicant',
+                                      foreign_key: :applicant_id,
                                       inverse_of: :applicant
-  has_many   :applied_grants,         through: :submissions,
+  has_many   :submissions,            through: :submission_applicants
+
+  has_many   :submitted_grants,       through: :submissions,
                                       source: :grant
```

Comment:
see previous comments. what are we trying to capture w/ the applicant concept? does the submitter capture that? does the statement "user has many `submitted_grants`" capture the meaning of applying to a grant?

Reply:
I think I would eliminate line 34-35. I think applied_grants captures what used to be submitted_grants.

Reply:
Shouldn't `has_many :applied_grants, though: :submission_applicants, [...]` be here? It got deleted.

Reply:
This was breaking with the new nesting and it wasn't being used, so I decided to remove it.